### DISCUSSION TITLE
The Notifications Bar annoys me and I don't like it

### DISCUSSION THREAD

**Discussion by ghost** [community/community#34596] [Other Feature Feedback, Questions, & Ideas]
Whenever I click a notification from `https://github.com/notifications` the site displays an annoying floating bar with no option of disabling it from appearing. [0]

This annoys me and I don't like it.

[0] 
![image](https://user-images.githubusercontent.com/623736/193061281-a73b4a48-1117-4253-82be-26ec75d6f1ee.png)

**Comment by ghost** [comment]
Just put all the Notifications actions (Done, Unsubscribe, Mark as unread and Save) on the side IMO, here:

![image](https://user-images.githubusercontent.com/623736/193062975-562c69a9-ff92-4677-ac6d-dea22d337876.png)

**Comment by 101100** [comment]
Reloading the page or moving to a new commit also re-opens this annoying toolbar. I would love to be able to remove it permanently.
